bitnami/spring-cloud-dataflow/Chart.yaml

```diff
@@ -39,4 +39,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-spring-cloud-dataflow
   - https://github.com/bitnami/bitnami-docker-spring-cloud-skipper
   - https://dataflow.spring.io/
-version: 4.1.6
+version: 4.1.7
```

Comment:
As you are introducing a new parameter, instead of the patch version the minor version should be increased. Hence this should be `4.2.0`

Reply:
Hi @rafariossaa 

Thanks for the feedback - fixed.